src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private UserService userService;
+
+    public AuthenticationService() {
+        this.userService = new UserService();
+    }
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return user != null && user.getEmail().equals(email) && user.getPassword().equals(password);
```

Comment:
we can also say that email check is redundant here, because we have found that user by email